src/interfaces/assistants_web/src/components/Agents/AgentRightPanel.tsx

```diff
@@ -1,17 +1,18 @@
 'use client';
 
 import { Transition } from '@headlessui/react';
+import { uniqBy } from 'lodash';
 import { useMemo, useState } from 'react';
 
 import { CitationsTab } from '@/components/Agents/CitationsTab';
 import { IconButton } from '@/components/IconButton';
 import { Banner, Button, Icon, Switch, Tabs, Text, Tooltip } from '@/components/Shared';
-import { TOOL_GOOGLE_DRIVE_ID } from '@/constants';
+import { TOOL_GOOGLE_DRIVE_ID, TOOL_READ_DOCUMENT_ID, TOOL_SEARCH_FILE_ID } from '@/constants';
```

Comment:
side note: could be nice to take a look at the constants and organize into data sources vs tools

Reply:
yes agree it is growing quite a bit